lib/inspec/resource.rb

```diff
@@ -89,6 +89,8 @@ def self.validate_resource_dsl_version!(version)
 require 'resources/docker_image'
 require 'resources/docker_container'
 require 'resources/etc_group'
+require 'resources/etc_hosts_allow_deny'
+require 'resources/etc_hosts'
```

Comment:
Line 93 has nothing to do with this PR. I think this is likely from a bad merge conflict resolution. Please rebase on master and ensure your changes are only related to your PR focus.

Reply:
Sounds good, Im having a little trouble Ill need to talk to you or Aaron tomorrow if you have time?

Reply:
I'm traveling for work today and tomorrow, but I'd be happy to pair with you on Wednesday.